recipes/rapidshapes/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "2.3.2" %}
+{% set packagename = "rapidshapes" %}
+{% set sha256 = "d3e6cc8d2ecd31d4764b41fea589b98c637ff227d7ab3dcfe66f2ad3f24dece5" %}
+
+package:
+  name: {{ packagename }}
+  version: 2.1.0
```

Comment:
2.3.2 ? i.e. `{{ version }}`

Reply:
It is confusing but I think I understand your idea - the fold-grammars is version 2.3.2 but rapidshapes is 2.1.0.
OK ...

Reply:
no, this is the version of upstream repo, which served multiple conda packages, e.g. rnashapes, pkiss, knotinframe, ...

Reply:
It is confusing.
I think the conversation in the PR for [knotinframe](https://github.com/bioconda/bioconda-recipes/pull/49226) is the most up-to-date why confuses people and bots.